src/components/LanguagePicker/useLanguagePicker.tsx

```diff
@@ -106,10 +108,9 @@ export const useLanguagePicker = (
               (dataItem!.words.approved / dataItem!.words.total) * 100
             ) || 0
 
-      if (progressData.length === 0)
-        throw new Error(
-          "Missing translation progress data; check GitHub action"
-        )
+      if (progressData.length === 0) {
+        console.warn("Missing translation progress data; check GitHub action")
+      }
```

Comment:
Looking at this one again, I think we can probably keep this error. It should never throw as long as there is translation progress data available... if not, we should let the build break. cc: @pettinarip Curious your thoughts

Reply:
I've to disagree on that. This condition is inside a `useEffect`. It will never be checked at build time/server side. If it is ever checked, then it will be on client side, creating bad ux since it will make the app crash (unless we are catching that error and failing gracefully).

However, I hear you, it would be nice to have that type of check at build time.

And the change to use `console.warn` is ok I guess but its not doing anything. Shouldn't  we return something else here? given that in the next line we are accessing it `progressData[0].words.total`.

Reply:
Okay yeah, that makes sense, thanks

> Shouldn't we return something else here? given that in the next line we are accessing it progressData[0].words.total.

I think we should... I can update this to return early in this case, and pass everything but the `approvalProgress` and `wordsApproved`. We can also use `localeOption` (`Lang` type) as the fallback for the source or target names... this means a language that failed this for whatever reason would just show the language code instead of an empty string.

Will push a patch